cni/pkg/taint/taint.go

```diff
@@ -0,0 +1,154 @@
+package taint
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"gopkg.in/yaml.v2"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	client "k8s.io/client-go/kubernetes"
+
+	"istio.io/pkg/log"
+)
+
+const (
+	TaintName = "NodeReadiness"
+)
+
+type ConfigSettings struct {
+	Name          string `yaml:"name"`
+	Namespace     string `yaml:"namespace"`
+	LabelSelector string `yaml:"selector"`
+}
+
+func (config ConfigSettings) ToString() string {
```

Comment:
This doesn't add value in my opinion, and should be called String() if we really want a custom format (see https://tour.golang.org/methods/17)

Reply:
Used String()